claim-check-interceptors-azure-8/src/main/java/se/irori/kafka/claimcheck/azurev8/AzureBlobStorageClaimCheckBackendV8.java

```diff
@@ -64,6 +65,11 @@ public ClaimCheck checkIn(ProducerRecord<byte[], byte[]> largeRecord) {
     return new ClaimCheck(blockBlobReference.getUri().toString());
   }
 
+  @Override
+  public ClaimCheck checkInStreaming(String topic, InputStream payload, long payloadSize) {
+    throw new UnsupportedOperationException("Not implemented for Azure V8 backend");
```

Comment:
Is it much work to do the v8 implementation as well?

Reply:
I haven't looked, but I think eventually that whole implementation should be removed since v8 SDK isn't kept very up to date. But perhaps we should have some alternative implementation for people who are getting dependency collisions, perhaps
https://jclouds.apache.org/

Reply:
Turned out to be easy to implement so I just did it for v8